salt/engines/kujira_parser/redisDbLib.py

```diff
@@ -0,0 +1,44 @@
+import redis
```

Comment:
This module is not implemented by you.
Commit & push code developed only by you.